assets/wizards/analytics/views/configuration/style.scss

```
@@ -0,0 +1,37 @@
+@import '~@wordpress/base-styles/colors';
+
+.newspack__analytics-configuration {
+	th {
+		text-align: left;
+		border-bottom: 1px solid $light-gray-500;
+	}
+	table {
+		width: 100%;
+		margin-bottom: 4em;
+		border-spacing: 0;
+	}
+	tbody {
+		td {
+			padding-top: 4px;
```

I found after I button mashed and added a few dimensions, they ran together a bit; I think adding borders between and a bit more spacing could help, like: 

```suggestion
			padding: 4px 0;
			border-top: 1px solid $light-gray-500;
```

(This also ends up making a stronger border underneath the header, so it's still visually distinct from the list!)